packages/ui-extensions/src/surfaces/admin/components/CustomerSegmentTemplate/CustomerSegmentTemplate.ts

```diff
@@ -7,20 +7,32 @@ import {createRemoteComponent} from '@remote-ui/core';
 type CustomerStandardMetafieldDependency = 'facts.birth_date';
 
 export interface CustomerSegmentTemplateProps {
-  /* Localized title of the template. */
+  /**
+   * Localized title of the template.
+   */
   title: string;
-  /* Localized description(s) of the template. */
+  /**
+   * Localized description(s) of the template.
+   */
   description: string | string[];
-  /* Code snippet to render in the template with syntax highlighting. */
+  /**
+   * Code snippet to render in the template with syntax highlighting.
+   */
   query: string;
-  /* Code snippet to insert in the segment editor. If missing, `templateQuery` will be used. */
+  /**
+   * Code snippet to insert in the segment editor. If missing, `templateQuery` will be used.
+   */
   queryToInsert?: string;
-  /* List of customer standard metafields or custom metafields used in the template's query. */
+  /**
+   * List of customer standard metafields or custom metafields used in the template's query.
+   */
   dependencies?: {
     standardMetafields?: CustomerStandardMetafieldDependency[];
     customMetafields?: string[];
   };
-  /* ISO 8601-encoded date and time string. A "New" badge will be rendered for recently introduced templates. */
+  /**
+   * ISO 8601-encoded date and time string. A "New" badge will be rendered for recently introduced templates.
```

Comment:
how is recency determined? is it by time period, or something else? 

```suggestion
   * ISO 8601-encoded date and time string. A **New** badge is rendered for recently introduced templates.
```